addons/stock/report/product_templates.xml

```diff
@@ -21,10 +21,10 @@
 ^FO600,100,1
 ^CI28
 <t t-if="product.currency_id.position == 'after'">
-^A0N,66,48^FH^FD<t t-esc="product.list_price" t-options='{"widget": "float", "precision": 2}'/><t t-esc="product.currency_id.symbol"/>^FS
+^A0N,66,48^FH^FD<t t-esc="product.lst_price" t-options='{"widget": "float", "precision": 2}'/><t t-esc="product.currency_id.symbol"/>^FS

```

Comment:
Will raise a traceback when generated from a `product.template`, as `lst_price` doesn't exist in this case.